### DISCUSSION TITLE
Google Drive Integration for Assets

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#16208] [Q&A / Support]
Hi All,

How to integrate Google Drive like AWS s3 bucket?
If a user uploads assets from Pimcore Admin then those Assets will be stored on Google Drive in the proper folder.

It's similar to using the AWS S3 bucket, here we need to use Google Drive instead AWS S3 bucket.

**Comment by kingjia90** [comment]
Probably you need something like https://packagist.org/packages/modernmcguire/flysystem-google-drive and changing the configuration on the flysystem yaml https://github.com/pimcore/pimcore/blob/512cc910427e8603572508da1d782b8c6a0ebb5a/.github/ci/files/minio-flysystem.yaml#L16-L21 but it's not something that i've tried and is not tested/supported
